src/routes/tftRoutes.js

```diff
@@ -5,34 +5,69 @@ import riotApiClient from '../services/riotApiClient'
 function getTFTRoutes(){
     const router = express.Router()
     router.get('/v1/summoner/:summonerName', getSummonerHandler)
-    router.get('/v1/match/:summonerName', getRecentMatchInfo)
-    router.get('/v1/match/history/:summonerName', getMatchHistoryForSummoner)
-    //router.get('/v1/stats/:count/:summonerName', getAverageScores)
+    router.get('/v1/match/history/:summonerName/:matchId', getMatchInfo)
```

Comment:
My suggestion is to name the handler functions something like this, and you don't need a path parameter for summoner:
```suggestion
    router.get('/v1/match/history/:matchId', handleGetMatchDetail)
```